src/main/java/org/cbio/gdcpipeline/reader/ClinicalReader.java

```diff
@@ -65,10 +65,16 @@ public void open(ExecutionContext executionContext) throws ItemStreamException {
             throw new ItemStreamException("Empty Clinical Files list");
         }
 
+        List<String> skippedList = new ArrayList<>();
         for (String clinicalFile : clinicalFileNames) {
-            File file = new File(sourceDir + File.separator + cancer_study_id + File.separator + clinicalFile);
+            File file = new File(sourceDir + File.separator + clinicalFile);
             if (!file.exists()) {
-                throw new ItemStreamException(" Resource Error. File does not exists : " + file.getAbsolutePath());
+                if (LOG.isErrorEnabled()) {
```

Comment:
There are two cases : 
1. There are barcodes (or samples) that are extracted from biospecimen files but corresponding clinical files are not found during clinical step
2. There are barcodes ( or samples) that are extracted during clinical step that were not seen during biospecimen step.